src/tiny.ts

```diff
@@ -425,7 +426,8 @@ export class TinyPg {
             }
 
             const result = await new Promise<Pg.QueryResult>((resolve, reject) => {
-               query.callback = (err, res) => (err ? reject(err) : resolve(res))
+               // The type definition does not know .callback is a function.
```

Comment:
do we have the latest version of the pg types?

Reply:
Yes, latest is `7.14.3` at this time.